utility_analysis_new/utility_analysis.py

```diff
@@ -32,6 +32,7 @@ class UtilityAnalysisOptions:
     aggregate_params: pipeline_dp.AggregateParams
     multi_param_configuration: Optional[
         dp_engine.MultiParameterConfiguration] = None
+    partitions_sampling_prob: float = 1
 
     def __post_init__(self):
```

Comment:
Could we validate `0 < partitions_sampling_prob <= 1` here?

Reply:
Yes, thanks